src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/WindowsRemoteAuthenticationProvider.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.ProjectSystem.Debug;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.PropertyPages
+{
+    /// <summary>
+    /// Implementation of IRemoteAuthenticationProvider for the Windows authentication type.
+    /// </summary>
+    [Export(typeof(IRemoteAuthenticationProvider))]
+    [AppliesTo(ProjectCapabilities.AlwaysApplicable)]
+    internal class WindowsRemoteAuthenticationProvider : IRemoteAuthenticationProvider
+    {
+        [ImportingConstructor]
+        public WindowsRemoteAuthenticationProvider(UnconfiguredProject _) // force MEF scope
+        {
+        }
+
+        public string Name => Resources.WindowsAuth;
+
+        public Guid PortSupplier => Guid.Empty;
```

Comment:
Is this not 708C1ECA-FF48-11D2-904F-00C04FA302A1?

Reply:
I got confused by [this line](http://index/?query=UAPDebuggerExtensions&rightProject=Microsoft.VisualStudio.UniversalLibrary&file=Microsoft%5CVisualStudio%5CUniversalLibrary%5CProjectFlavoring%5CDebug%5CUAPDebuggerExtensions.cs&line=20) but based on the code in [this method](http://index/?query=%60GetGuidPortSupplier&rightProject=Microsoft.VisualStudio.UniversalLibrary&file=Microsoft%5CVisualStudio%5CUniversalLibrary%5CProjectFlavoring%5CProjectConfig%5CUAPProjectFlavorCfg.cs&line=159) I think you're right, and in that method Universal uses No_Auth so I'm guessing there are some leaks to this abstraction.

Reply:
Okay, dug in a bit more. This should be passing 708C1ECA to the debugger, but if we pass that to the Find dialog it will throw 🤦‍♂️

Reply:
Why does the dialog throw this? Can we fix it, as it just simplifies this whole this.

Reply:
It's trivial to fix, but I've reached out to confirm the expected behaviour of these guids as the Universal auth mode also uses two different guids at times.